test/try-server.ts

```diff
@@ -17,17 +17,21 @@ import {Datastore} from '../src';
 
 import {startServer} from '../mock-server/datastore-server';
 
-describe('Try server', () => {
-  it.skip('should try to connect to the running server', done => {
+describe.skip('Try server', () => {
+  it('should try to connect to the running server', done => {
     startServer(async () => {
-      const datastore = new Datastore({
-        namespace: `${Date.now()}`,
-        apiEndpoint: 'localhost:50051',
-      });
-      const postKey = datastore.key(['Post', 'post1']);
-      const query = datastore.createQuery('Post').hasAncestor(postKey);
-      const allResults = await datastore.runQuery(query);
-      done();
+      try {
+        const datastore = new Datastore({
+          namespace: `${Date.now()}`,
+          apiEndpoint: 'localhost:50051',
+        });
+        const postKey = datastore.key(['Post', 'post1']);
+        const query = datastore.createQuery('Post').hasAncestor(postKey);
+        const allResults = await datastore.runQuery(query);
+        done();
+      } catch (e) {
+        done(e);
+      }
```

Comment:
The only real change is to make the test runner throw an error if the script throws an error instead of failing silently